data/console/ansible/roles/test/meta/main.yaml

```
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  role_name: test
+  namespace: openQA
+  author: Pavel Dostál
+  description: Test role
+  license: FSFAP
+  min_ansible_version: "2.9"
+  platforms:
+    - name: opensuse
```

```suggestion
    - name: opensuse
      versions:
        - 15.3
        - 15.4
        - tumbleweed
    - name: sles
      versions:
        - 12SP3
        - 12SP4
        - 12SP5
        - 15
        - 15SP1
        - 15SP2
        - 15SP3
        - 15SP4
```

For the suggested scheme see e.g. https://galaxy.ansible.com/api/v1/platforms/?search=sles

---

@grisu48 you really want to introduce a secluded definition file that needs to be updated with every SLE+Leap service pack?

---

I would actually remove the `platforms` section for exactly this reason @okurz 
But if it is there, then we should do it properly.

---

@pdostal your call - feel free to remove the `plaforms` section as a whole if you agree that we should not have this maintenance burden. The choice is up to you, I'm fine with your current approach as well :slightly_smiling_face: